ledger/sync/src/ledger_sync/ledger_sync_service.rs

```diff
@@ -34,27 +34,19 @@ const DEFAULT_GET_BLOCKS_TIMEOUT: Duration = Duration::from_secs(30);
 const DEFAULT_GET_TRANSACTIONS_TIMEOUT: Duration = Duration::from_secs(30);
 
 pub struct LedgerSyncService<L: Ledger, BC: BlockchainConnection, TF: TransactionsFetcher> {
-    /// Local ledger database.
```

Comment:
Why did some comments get removed and one got to stay?

Reply:
Just a caffeinated impulse. Some of the comments felt like tautologies, e.g.:

```
/// Transactions Fetcher.	
    transactions_fetcher: Arc<TF>
```